scripts/checksum

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+var shell = require('shelljs');
```

Comment:
Is there a reason we're using `var`?

Reply:
These scripts do not go through babel. I wanted to make sure there are no problems with older versions of node (if any). Hence all scripts use ES5 syntax - `function`, `var`, `for` loops etc.

Reply:
In near future I will enable test matrix and run tests on multiple node versions.

Reply:
I don't think this is a good reason. I would suggest linking up with [node.green](https://node.green/) and target LTS (10.15.3) and above, meaning you can use a _whole lot more_ than ES5. 

The reason for this is that we don't want to set precedent of now being okay with forcefully using old (and even deprecated) modules and usages. Once they're in there, it's another hurldle to get out. The lowest node for this track is LTS anyway, so this shouldn't be a current problem.

Reply:
> The lowest node for this track is LTS anyway.

This is not something to feel great about IMO. Most users might have older versions of Node installed (work projects etc. - lot of them might not be aware of `nvm`) and we have to make sure the exercises & tests work for them - hence the test matrix.

The best solution will be to use babel over these scripts as well - which I assume will get best of both worlds - keep our code on shiny latest ES and make sure it runs on old node versions.

Reply:
Ok. I think running babel on these scripts is a compromise I can definitely live with, especially since it can target the _current user node_, so it will NOT generate `var` if on LTS :).

The "most users are on older versions" argument is what's causing security issues in government software still targeting windows 95 and I think not one we should uphold for _maintaining_ or _contributing_ to this repository. It means that we're okay with users running versions with security holes, issues, deprecated or removed functionality. But that's the last I'll say about it. If our endgoal is as you say: babel + test matrix, I am sufficiently happy ❤️

Reply:
We never mention leave alone ask users to use old platforms. But we should also make sure our software doesn't break on them. Hopefully this will be good for new-comers who don't get confused and discouraged by unrelated errors.

Reply:
> But we should also make sure our software doesn't break on them.

Don't agree, but I think that's where our opinion differs :)